app/models/post.rb

```diff
@@ -1,2 +1,15 @@
 class Post < ApplicationRecord
+    belongs_to :author, class_name: 'User'
+    has_many :likes
+    has_many :comments
+
+    after_save :update_posts_counter
+
+    def update_posts_counter
```

Comment:
Great job @demix007 🤝

- [ ] [Optional] You may also make the methods `update_posts_counter`, `update_likes_counter`, and `update_comments_counter` private.